src/video/cocoa/SDL_cocoamessagebox.m

```diff
@@ -55,28 +55,18 @@ - (id)initWithParentWindow:(SDL_Window *)window
 - (void)showAlert:(NSAlert *)alert
 {
     if (nswindow) {
-#ifdef MAC_OS_X_VERSION_10_9
         if ([alert respondsToSelector:@selector(beginSheetModalForWindow:completionHandler:)]) {
             [alert beginSheetModalForWindow:nswindow
                           completionHandler:^(NSModalResponse returnCode) {
-                            self->clicked = returnCode;
+                            [NSApp stopModalWithCode:returnCode];
                           }];
-        } else
-#endif
-        {
-#if MAC_OS_X_VERSION_MIN_REQUIRED < 1090
```

Comment:
This #if was there to avoid the code being compiled (and therefore avoid a deprecation warning when compiling) when the min runtime target is set to an OS version where the function is deprecated. The above codepath will always be hit instead in that situation, so preventing this codepath's code from being compiled shouldn't be an issue.